INSTALL.md

```diff
@@ -2,10 +2,19 @@
 
 Please follow each part of this documentation in order to run your own instance of Saskatoon.
 
+## Overview
+1. [Install requirements](#requirements).
+2. [Prepare `.env` settings](#.env-settings).
+3. [Set up local database](#local-database).
+4. [Set up Django](#django-setup).
+5. [Load initial data (fixtures)](#loading-initial-data-(fixtures)).
+
 ## Requirements
 
-1. MySQL client
+1. Python 3.9 (included in Debian Bullseye)
```

Comment:
you missed an opportunity to start the list at 0.     ;P

Reply:
i thought about it! but starting md lists at 1 *feels* right lol